tests/custom-elements/card.pug

```diff
@@ -0,0 +1,281 @@
+extends ../layout.pug
+
+block scripts
+  script(type="module" src='../../js/ce/ux-chi-ce/ux-chi-ce.esm.js')
+  script(src='../../js/chi.js')
+  script
+    | document.addEventListener('DOMContentLoaded', function() {
```

Comment:
You can add scripts at the bottom, take a look to other examples , e.g: `carousel`